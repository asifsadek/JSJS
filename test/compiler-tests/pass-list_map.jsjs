val a = [1, 2, 3, 4, 5];

List.print_list(List.map(/\(x:num): num => x * x, a));

List.print_list(List.map(/\(x: num): string => if (x%2 == 0) then "EVEN" else "ODD", a));


