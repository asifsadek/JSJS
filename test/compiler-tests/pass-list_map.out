1
4
9
16
25
ODD
EVEN
ODD
EVEN
ODD
